pkg/trace/stats/concentrator.go

```diff
@@ -31,34 +33,33 @@ type Concentrator struct {
 	// It means that we can compute stats only for the last `bufferLen * bsize` and that we
 	// wait such time before flushing the stats.
 	// This only applies to past buckets. Stats buckets in the future are allowed with no restriction.
-	bufferLen int
-
-	In  chan []Input
-	Out chan []Bucket
-
-	exit   chan struct{}
-	exitWG *sync.WaitGroup
-
-	buckets map[int64]*RawBucket // buckets used to aggregate stats per timestamp
-	mu      sync.Mutex
+	bufferLen     int
+	In            chan []Input
```

Comment:
If you've merged all of this together, let's create two groups:
* exported fields at the top, then one empty line and
* unexported fields